corehq/apps/api/odata/views.py

```diff
@@ -19,10 +20,11 @@ def get(self, request, domain):
             '@odata.context': absolute_reverse('odata_meta', args=[domain]),
             'value': [
                 {
-                    'name': 'Cases',
+                    'name': case_type,
                     'kind': 'EntitySet',
-                    'url': 'Cases',
+                    'url': case_type,
                 }
+                for case_type in get_case_types_for_domain_es(domain)
```

Comment:
nice. does PowerBI actually honor the service response? I never found a way to get tableau to use it.

Reply:
I'm not sure if it's useful for PowerBI or Tableau, but seemed worth including as it's part of the specification and simple to do.